app/services/call_creator.rb

```diff
@@ -10,9 +11,7 @@ def run
                      member_id: @params[:member_id],
                      member_phone_number: @params[:member_phone_number],
                      target_index: @params[:target_index])
-    if @call.save
-      place_call
-    end
+    place_call if @call.save
```

Comment:
I wouldn't take the liberty of changing the style of your code. It was rubopcop. Take it up with him!